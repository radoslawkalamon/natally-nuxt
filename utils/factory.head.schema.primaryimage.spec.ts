import { factoryHeadSchemaPrimaryimage } from '@/utils/factory.head.schema.primaryimage'

describe('Utils / Factory / Head / Schema / Primary image', () => {
  test('shall match snapshot | { path }', () => {
    const args = {
      path: '/test/path/'
    }
    expect(factoryHeadSchemaPrimaryimage(args)).toMatchSnapshot()
  })

  test('shall match snapshot | { path, imageOpenGraph }', () => {
    const args = {
      path: '/test/path/',
      imageOpenGraph: '/images/test.path.opengraph.png'
    }
    expect(factoryHeadSchemaPrimaryimage(args)).toMatchSnapshot()
  })
})
