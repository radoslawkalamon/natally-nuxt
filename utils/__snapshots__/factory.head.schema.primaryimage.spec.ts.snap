// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Utils / Factory / Head / Schema / Primary image shall match snapshot | { path } 1`] = `
Object {
  "@id": "https://169cm.pl/test/path/#primaryimage",
  "@type": "ImageObject",
  "contentUrl": "https://169cm.pl/images/common.opengraph.png",
  "height": 630,
  "hid": "schema:ImageObject",
  "inLanguage": "pl-PL",
  "url": "https://169cm.pl/images/common.opengraph.png",
  "width": 1200,
}
`;

exports[`Utils / Factory / Head / Schema / Primary image shall match snapshot | { path, imageOpenGraph } 1`] = `
Object {
  "@id": "https://169cm.pl/test/path/#primaryimage",
  "@type": "ImageObject",
  "contentUrl": "https://169cm.pl/images/test.path.opengraph.png",
  "height": 630,
  "hid": "schema:ImageObject",
  "inLanguage": "pl-PL",
  "url": "https://169cm.pl/images/test.path.opengraph.png",
  "width": 1200,
}
`;
